### ISSUE TITLE
Chips: JAWS doesn't read input text when chips are present

### ISSUE THREAD

**Issue by karthikkoundinya** [angular/components#20360]
#### Reproduction

Use StackBlitz to reproduce issue: 
https://stackblitz.com/edit/angular-1oqjvx?file=src/app/autocomplete-auto-active-first-option-example.ts

Steps to reproduce:
1. Please use mat-chip-List along with mat-input and take look above code link for more details 
2.	Enter the one from the dropdown add 2-3 mat-chip
![image](https://user-images.githubusercontent.com/10532432/90632029-5846ad80-e241-11ea-8765-e3361ef00829.png)
3.	JAWS screen reader is not able read the inputs entered with one or more mat-chip when we using read All in the screen reader[Use JAWS screen reader to read the screen, use shortcut  insert+ down-arrow key  button to Read all the screen]
4. This issue is applicable in case of read all (Insert+ Down Key)

 #### Expected Behavior
Its should read all mat-chip added with input entered using mat-autocomplete 
What behavior were you expecting to see?
Screen reader should be able to read the mat-chip  entered with input field when screen is reading with Read All screen 

#### Actual Behavior

What behavior did you actually see?
Jaws Screen reader is not able to read the mat-chips entered with input field its completely ignoring the mat-chips steps and continue reading with next steps when screen is reading with Read All screen

**Comment by jelbourn** [comment]
Without directly looking into this, I suspect it will be fixed with our MDC-based implementation of chips since we change the interaction pattern to one that works much better for the input + chips pattern.

**Comment by karthikkoundinya** [comment]
Thank you jelbourn 
Can you please provide me some more details on MDC-based with similar example

**Comment by nikhil4326** [comment]
Any updates on this issue?

**Comment by jelbourn** [comment]
The new chips implementation lives in `@angular/material-experimental` now; it will eventually graduate to the stable package.

**Comment by zarend** [comment]
Hello @karthikkoundinya ,

I cannot reproduce, and I believe this issue was fixed in v15. Does the issue persist for you?

Thanks,
Zach

**Comment by zarend** [comment]
Hello @karthikkoundinya ,

I can no longer reproduce, and I believe this issue was fixed in v15. Upgrade to version 15 to resolve this issue. We hope Angular Material meets your needs. If it does not, please reopen this issue or file a new issue.

-Zach
